Android.common.mk

```diff
@@ -160,4 +160,9 @@ LOCAL_CPPFLAGS += \
 	-DDISABLE_CURSOR_PLANE
 endif
 
+ifeq ($(strip $(BOARD_THREEDIS_UNDERRUN_WA)), true)
+LOCAL_CPPFLAGS += \
+        -DTHREEDIS_UNDERRUN_WA
```

Comment:
hmm We should enable this dynamically during run time. Probably, DrmDisplayManager, should check connected displays and set a flag in DisplayQueue, DisplayPlaneManager ? This way atleast we can use compression in supported combinations.

Reply:
Even with kernel investigation temporary should be the dynamic path like we do with cursor now. Where we try to disable it's usage when all three pipes are enabled. You can check that and do the same here.

Reply:
@kalyankondapally thanks for the comments.
Done the changes accrodingly